CONTRIBUTING.md

```diff
@@ -1,113 +1,29 @@
-# Contributing #
+Contributing
+============
 
-Information on contributing to this repo is in the [Contributing Guide](https://github.com/aspnet/Home/blob/dev/CONTRIBUTING.md) in the Home repo.
+The document covers the basic process for contributing to the [ASP.NET Core documentation site](https://docs.asp.net). See the [ASP.NET Core contributing guide](https://github.com/aspnet/Home/blob/dev/CONTRIBUTING.md) for information on contributing to ASP.NET Core source.
 
-The documentation is built using [Sphinx](http://sphinx-doc.org) and [reStructuredText](http://sphinx-doc.org/rest.html), and then hosted by [ReadTheDocs](http://aspnet.readthedocs.org).
+## Contributing process
 
-## Video: Getting Started ##
-[Watch a video](http://ardalis.com/contributing-to-asp-net-5-documentation) showing how to get started building the documentation locally.
+Small changes to one *.MD* file can be made in the browser by tapping the **Edit on GitHub** link in the upper right corner of the browser window. For new articles and changes involving more than one file:
 
-## Building the Docs ##
+* Open an issue describing the article you wish to write or update. Get approval before you invest too much time.
+* Fork the [aspnet/docs](https://github.com/aspnet/Docs/) repo and create a branch for your article.
+* Write or update your article using [DocFX](https://dotnet.github.io/docfx/tutorial/docfx_getting_started.html#2-use-docfx-as-a-command-line-tool) in [GitHub Flavored Markdown](https://guides.github.com/features/mastering-markdown/). Images go in the *\<article-name>/_static* folder. Code goes in the *\<article-name>/samples* folder. It's often helpful to review an existing article to use as an template (for example, *tutorials\first-web-api.md* and the assets in *tutorials\first-web-api*).
```

Comment:
That Docfx link isn't about writing or updating, it's about building the site. We should link to something about DFM, such as how to do snippets.
The first-web-api examples could be links.